### ISSUE TITLE
The default RPC endpoint is blocked. How can I get around this? 

### ISSUE THREAD

**Issue by shenhaoguangDayDayUp** [WalletConnect/web3modal#827]
**Describe the bug**
When I use @web3modal/html, I cannot access this interface https://explorer-api.walletconnect.com/v3/wallets?projectId=8e6b5ffdcbc9794bf9f4a1952578365b&page=1&entries=10&device=mobile,or https://rpc.walletconnect.com/v1/?chainId=eip155:5&projectId=8e6b5ffdcbc9794bf9f4a1952578365b.
But when I turn on VPN, I can access the interface

**SDK Version**
 - Version [2.0.0-rc.1]

**Screenshots**
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/17617821/212795569-9b3320fb-3e08-43e4-96b0-4a119415c828.png">
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/17617821/212796300-2cc16fb1-926b-43c0-8432-456bd7c481dc.png">

**Comment by 0xAsimetriq** [comment]
It is due to regional rules / firewalls. We are working on a solution and option to define new url's i.e. `.org` for these endpoints, to unblock these resources.

**Comment by 0xAsimetriq** [comment]
Closing in favor of https://github.com/WalletConnect/web3modal/issues/829
